warehouse/forklift/legacy.py

```diff
@@ -875,7 +875,13 @@ def file_upload(request):
                     lambda: request.POST["content"].file.read(8096), b""):
                 file_size += len(chunk)
                 if file_size > file_size_limit:
-                    raise _exc_with_message(HTTPBadRequest, "File too large.")
+                    raise _exc_with_message(
+                        HTTPBadRequest,
+                        "File too large." +
+                        "Limit for project {name} is {limit}MB".format(
+                            name=project.name,
+                            limit=file_size_limit,
```

Comment:
Note that `file_size_limit` is in bytes, not megabytes, so this is going to output something like:

> File is too large. Limit for project foobar is 62914560MB.

To convert bytes to megabytes, so you can just do:

```py
file_size_limit // (1024 * 1024)
```

Also, it'd be nice if the project name was in quotes!

Reply:
Just came accross this in the Travis job logs, wondered why the project had a size limit of 60TB 😆